uxarray/core/dataset.py

```diff
@@ -220,7 +220,7 @@ def from_dict(cls, data, **kwargs):
         )
 
     @classmethod
-    def from_structured(cls, ds: xr.Dataset, tol: Optional[float] = 1e-10):
+    def from_structured(cls, ds: xr.Dataset):
```

Comment:
This is not related to this PR, why is the tolerance removed?

Reply:
Noticed this when I was working on the code; this variable wasn't used / was leftover.